warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/NonShardedSplitsFile.java

```diff
@@ -131,12 +143,12 @@ public URI getUri() {
         }
     }
     
-    private static String createFileName(boolean isTrimmed) {
-        return (isTrimmed ? "trimmed_" : "full_") + SPLITS_FILE_NAME_PROPERTY_VALUE;
+    private static String createFileName(SplitsFileType splitsFileType) {
+        return (splitsFileType) + SPLITS_FILE_NAME_PROPERTY_VALUE;
     }
     
-    public static Path findSplitsFile(Configuration conf, Path[] filesToCheck, boolean isTrimmed) {
-        String fileName = createFileName(isTrimmed);
+    public static Path findSplitsFile(Configuration conf, Path[] filesToCheck, SplitsFileType splitsFileType) {
+        String fileName = createFileName(splitsFileType);
         if (filesToCheck != null) {
             for (Path cacheFile : filesToCheck) {
                 if (matchesFileName(fileName, cacheFile)) {
```

Comment:
Why are we looking for a matching file (i.e. endsWith(fileName)) as opposed to looking for an exact match?